lib/Storage/Google.php

```diff
@@ -60,18 +60,16 @@ public function __construct($params) {
 			&& isset($params['client_id']) && isset($params['client_secret'])
 			&& isset($params['token'])
 		) {
-			$this->client = new \Google_Client();
+			$config = [
+				'retry' => [
+					'retries' => 5
+				]
+			];
+			$this->client = new \Google_Client($config);
 			$this->client->setClientId($params['client_id']);
 			$this->client->setClientSecret($params['client_secret']);
 			$this->client->setScopes(['https://www.googleapis.com/auth/drive']);
 			$this->client->setAccessToken($params['token']);
-			// if curl isn't available we're likely to run into
```

Comment:
@PVince81 I think you'll need to do `json_decode($params['token'])` here as well since you have encode the token in first place